SMLHelper/Patchers/CraftDataPatcher.cs

```diff
@@ -149,7 +149,7 @@ private static void AddCustomTechDataToOriginalDictionary()
             {
                 Logger.Log($"Removing conflicting TechData entries from patching.{Environment.NewLine}" +
                            $"This is so the user will take notice and know to resolve the conflict.{Environment.NewLine}" +
-                           "Only one custom TechData may be added per TechType.");
+                           "Only one custom TechData may be added per TechType.", LogLevel.Debug);
```

Comment:
Let's promote this to be a `Warn` since we should be able to recover from this error.
Also, this will have to later be reconciled with the changes in #77